### ISSUE TITLE
How to export or represent coefficients of linear regression?

### ISSUE THREAD

**Issue by leaf918** [biolab/orange3#5898]
**What's wrong?**
I do wanna see the coiffcients of linear regression.
But ,there is only a dash line to datatable widge.

**How can we reproduce the problem?** 
![image](https://user-images.githubusercontent.com/170311/158512372-bfcd8e6d-8ea8-4ce0-880a-38d1a65565b1.png)

**What's your environment?**
3.31

**Comment by leaf918** [comment]
![image](https://user-images.githubusercontent.com/170311/158569523-9b624e1d-228f-4025-b826-5f0f1f03fb72.png)

work fine in another mode.

**Comment by ajdapretnar** [comment]
Yes, the second example is how you should use it. I will update the documentation to be more explicit about it.
